SpotifyAPI.Web/Models/Request/LibraryEpisodesRequest.cs

```diff
@@ -0,0 +1,20 @@
+namespace SpotifyAPI.Web
+{
+  public class LibraryEpisodesRequest : RequestParams
+  {
```

Comment:
The parameter `market` is missing here, see https://developer.spotify.com/documentation/web-api/reference/#endpoint-get-users-saved-episodes